metadrive/component/traffic_participants/base_traffic_participant.py

```diff
@@ -61,6 +61,14 @@ def add_body(self, physics_body):
         self._body.set_friction(0.)
         self._body.set_anisotropic_friction(LVector3(0., 0., 0.))
 
+    def standup(self):
```

Comment:
rename this function to set_posture?

Reply:
We can add a new API call "set_posture", and keep this "standup" here but calling our new API.

Reply:
No. we have set_pitch() and set_roll() already. This is a special function for bicycles or pedestrians to prevent them from falling down when contacting with terrain or colliding with other objects

Reply:
so could we call set_pitch and set_roll here?